### PR TITLE
Cog 1328 enable reset button on detail page

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
#### Which User story does this relate to (link)?
[COG-1328](https://osi-cwds.atlassian.net/browse/COG-1328)

#### Brief feature description
-RESET button in user profile page is not enabled after user makes changes in email or phone number fields.

### CODE REVIEW

File: app/javascript/selectors/detailSelector.test.js [hunk 1]
```diff
@@ -531,40 +531,34 @@ describe('selectors', () => {
...
-      expect(disableActionButton(state)).toEqual(true)
+      const state = getState({ phoneNumber: '333-444' })
+      expect(disableSaveButton(state)).toEqual(true)
     })
 
     it('return false when email & phone number is valid', () => {
       const state = getState({
         phoneNumber: '1234567891',
-        email: 'hello@gmail.com',
+        email: 'hel@gmail.com',
```
Comment (kelfink) [inline app/javascript/selectors/detailSelector.test.js]: why this change?  Not objecting to it, I just don't know if it's relevant.  Does changing the email from hello (possibly in the before state). to this  change the logic?

Comment (Sushmareddy-OSI) [reply app/javascript/selectors/detailSelector.test.js]: Uhh No..., no logic this is accidental.
